### PR TITLE
First push

### PR DESCRIPTION
**Pull request by cgberlin**

### CODE REVIEW

**Comment by shellbell22** [comment]
Can you please push this to gh-pages so we can test it?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      <h1 id = "main-title">Pirate Bartender</h1>
+      <div class = "main-container">
+        <img src = "images/pirate.gif" alt = "pirate dog gif" />
+        <h2 id = "questions"></h2>
+        <div class = "buttons">
+          <button class = "answer">Yes</button>
+          <button class = "answer">No</button>
+        </div>
+      </div>
+
```
Comment (shellbell22) [inline index.html:22]: try to do the same amount of spacing between elements

**Comment by cgberlin** [comment]
Just pushed staging2 branch to gh-pages.

File: js/model.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+var questionCount = 0;
+
+
+var createQuestions = function(questions) {
```
Comment (shellbell22) [inline js/model.js:5]: The name of a class should start with upper case:(https://en.wikipedia.org/wiki/Naming_convention_(programming)#JavaScript)  
Class names should be nouns:
(http://www.oracle.com/technetwork/java/codeconventions-135099.html)

**Pull request closed without merging**